src/test/scala/mesosphere/marathon/core/launchqueue/impl/TaskLauncherActorTest.scala

```diff
@@ -129,7 +129,9 @@ class TaskLauncherActorTest extends AkkaUnitTest with Eventually {
       Mockito.when(instanceTracker.instancesBySpecSync).thenReturn(InstanceTracker.InstancesBySpec.forInstances(instances))
       val launcherRef = createLauncherRef()
       rateLimiterActor.expectMsg(RateLimiterActor.GetDelay(f.app.configRef))
-      rateLimiterActor.reply(RateLimiter.DelayUpdate(f.app.configRef, Some(clock.now())))
+      val mockedDelay = mock[Delay]
+      Mockito.when(mockedDelay.deadline).thenReturn(clock.now())
+      rateLimiterActor.reply(RateLimiter.DelayUpdate(f.app.configRef, Some(mockedDelay)))
```

Comment:
I am using `None` all through out the class wherever delay is needed because It defaults to `clock.now()` in `TaskLauncherActor`. I will fix this file based on how the conversation at https://github.com/mesosphere/marathon/pull/6700/files#r240594954 gets resolved.